docs/2.guide/1.directory-structure/1.app/1.plugins.md

```diff
@@ -235,7 +235,7 @@ export {}
 ```
 
 ::note
-If you are using WebStorm, you may need to augment `@vue/runtime-core` until [this issue](https://youtrack.jetbrains.com/issue/WEB-59818/VUE-TypeScript-WS-PS-does-not-correctly-display-type-of-globally-injected-properties) is resolved.
+If you are using WebStorm, you may need to augment `@vue/runtime-core` until [this issue](https://youtrack.jetbrains.com/issue/WEB-59818/VUE-TypeScript-WS-PS-does-not-correctly-display-type-of-globally-injected-properties) is resolved. (resolved from v2025.1)
```

Comment:
We should either remove this, or change the warning fully (e.g., "in the past you had to..."